docs/streams/developer-guide/config-streams.html

```diff
@@ -307,12 +300,12 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
               set by the user or all serdes must be passed in explicitly (see also default.key.serde).</td>
             <td><code class="docutils literal"><span class="pre">null</span></code></td>
           </tr>
-          <tr class="row-even"><td>default.windowed.key.serde.inner (Deprecated. Use windowed.inner.class.serde instead.)</td>
+          <tr class="row-even"><td>default.windowed.key.serde.inner</td>
```

Comment:
Why do we remove the deprecation statement?

Reply:
again, change coming in from trunk